scripts/setup_integration/setup_integration_env.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
I would recommend moving this up a level to `scripts/setup_integration_env.sh` putting it at a different level than the scripts it serves as a wrapper for. Of course, that messes with tab completion when running manually, but that's probably a lesser concern.